go.mod

```diff
@@ -233,3 +233,5 @@ require (
 )
 
 replace github.com/gomodule/redigo v2.0.0+incompatible => github.com/gomodule/redigo v1.8.5
+
+replace github.com/minio/madmin-go => ../madmin-go
```

Comment:
You have now `v1.1.22` with the API changes etc @poornas please pull `go get github.com/minio/madmin-go@v1.1.22`